File: test/fixtures/matlab/post_form.m
%% Web Access using Data Import and Export API
% This is not possible with the webread/webwrite API

%% HTTP Interface
import matlab.net.*
import matlab.net.http.*
import matlab.net.http.io.*

uri = URI('http://localhost:28139/post-to-me.php');
body = MultipartFormProvider('username', FormProvider('davidwalsh'), 'password', FormProvider('something'));
response = RequestMessage('post', [], body).send(uri.EncodedURI);

File: test/fixtures/dart/post_form.dart
import 'package:http/http.dart' as http;

void main() async {
  var url = Uri.parse('http://localhost:28139/post-to-me.php');
  var res = await http.MultipartRequest('POST', url)
    ..fields['username'] = 'davidwalsh'
    ..fields['password'] = 'something'
  if (res.statusCode != 200) throw Exception('http.post error: statusCode= ${res.statusCode}');
  print(res.body);
}
